<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Atom on miauhaus.log</title>
    <link>https://log.tuxedokatze.com/tags/atom/</link>
    <description>Recent content in Atom on miauhaus.log</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>ja</language>
    <copyright>&amp;copy;2017 tuxedocat</copyright>
    <lastBuildDate>Wed, 06 May 2015 00:00:00 +0000</lastBuildDate>
    
	<atom:link href="https://log.tuxedokatze.com/tags/atom/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>UnicodeEncodeError when running python scripts in Atom.io</title>
      <link>https://log.tuxedokatze.com/post/2015-05-06-atom-python-script/</link>
      <pubDate>Wed, 06 May 2015 00:00:00 +0000</pubDate>
      
      <guid>https://log.tuxedokatze.com/post/2015-05-06-atom-python-script/</guid>
      <description>&lt;h2 id=&#34;何&#34;&gt;何?&lt;/h2&gt;

&lt;p&gt;Atomのscript-pluginでpython実行時にsys.stdoutのencodingがUS-ASCIIになっていたので、&lt;code&gt;print()&lt;/code&gt;などで&lt;code&gt;UnicodeEncodeError&lt;/code&gt;が出る。&lt;/p&gt;

&lt;p&gt;&lt;/p&gt;</description>
    </item>
    
  </channel>
</rss>